### PR TITLE
Add los.direct.gov.uk to transition

### PR DESCRIPTION
**Pull request by benthorner**
https://govuk.zendesk.com/agent/tickets/3472971

### CODE REVIEW

File: data/transition-sites/directgov_los.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+site: directgov_los
+whitehall_slug: government-digital-service
```
Comment (andrewgarner) [inline data/transition-sites/directgov_los.yml]: I think this should be DWP. The previous site was branded as a service by "jobcentreplus Department for Work and Pensions" – https://web.archive.org/web/20180920044754/http://los.direct.gov.uk

```suggestion
whitehall_slug: department-for-work-pensions
```

File: data/transition-sites/directgov_los.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+homepage: https://www.gov.uk/government/organisations/government-digital-service
+tna_timestamp: 20181120000000
```
Comment (andrewgarner) [inline data/transition-sites/directgov_los.yml]: How did you find this timestamp? I think it should be `20180920044754` from looking at https://web.archive.org/web/*/http://los.direct.gov.uk

```suggestion
tna_timestamp: 20180920044754
```

File: data/transition-sites/directgov_los.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+site: directgov_los
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/organisations/government-digital-service
```
Comment (andrewgarner) [inline data/transition-sites/directgov_los.yml]: The README says this field should be:

> URL for new site, used to redirect '/'. Must include the 'http' or 'https'.

So shouldn't this be as follows:

```suggestion
homepage: https://find-your-nearest-jobcentre.dwp.gov.uk
```

**Comment by andrewgarner** [comment]
Looking at some of the previous configs I think we should add these additional attributes:

```yaml
homepage_title: Local office search
css: directgov
```

**Comment by benthorner** [comment]
@andrewgarner oops - forgot to actually redirect :-).

File: data/transition-sites/directgov_los.yml (hunk 2 above)
Comment (benthorner) [reply data/transition-sites/directgov_los.yml]: I tried, but might have typed it wrong - it often doesn't work and isn't necessary for anything. I'll update it though 👍 .

File: data/transition-sites/directgov_los.yml (hunk 1 above)
Comment (benthorner) [reply data/transition-sites/directgov_los.yml]: OK. I guess they might want to look at the analytics.

File: data/transition-sites/directgov_los.yml (hunk 3 above)
Comment (benthorner) [reply data/transition-sites/directgov_los.yml]: Fair enough - this is auto-populated in a rather unhelpful way by the rake task :-|.

**Pull request merged**